docling_core/types/doc/document.py

```diff
@@ -656,6 +655,15 @@ def export_to_document_tokens(
         return body
 
 
+class CodeItem(TextItem):
```

Comment:
I assume this should then also be included into the `ContentItem` tagged union.

Reply:
yes it should be.